pylsp_mypy/plugin.py

```diff
@@ -261,6 +262,7 @@ def pylsp_lint(
                     errors.strip(),
                 )
                 mypy_api.run_dmypy(["kill"])
+                mypy_api.run_dmypy(["start"])
```

Comment:
Nitpicking :)

```suggestion
                mypy_api.run_dmypy(["restart"])
```
`dmypy --help` states
```
...
restart           Restart daemon (stop or kill followed by start)
...
```